CONTRIBUTING.md

```diff
@@ -171,6 +171,16 @@ To format the documentation markdown and its code snippets, use:
 npm run markdownlint-fix
 ```
 
+## Writing TSDoc Comments
+
+When commenting, consider the following:
+
+- If there is a new release which hasn't been given any tag and its throwing some warning then mark that as `@public`
```

Comment:
I think we could change this to something like "Every new method needs to have either `@public` or `@internal` added depending on if it is part of the public API".

Reply:
Sure, I will replace it